sentry/src/routes/channel.rs

```diff
@@ -846,4 +857,86 @@ mod test {
             balances.spenders.get(&CREATOR),
         );
     }
+
+    #[tokio::test]
+    async fn get_channels_list() {
+        let mut app = setup_dummy_app().await;
+        app.config.channels_find_limit = 2;
+
+        let channel = Channel {
+            leader: IDS[&LEADER],
+            follower: IDS[&FOLLOWER],
+            guardian: *GUARDIAN,
+            token: "0x6B175474E89094C44Da98b954EedeAC495271d0F".parse()
```

Comment:
Are these tokens DAI addresses in different chains?

Reply:
This one is `chain."Ethereum Mainnet".token.DAI` while the other one is DAI on Goerli. I will clarify this in the expect.